src/ProgressOnderwijsUtils/Data/ParameterizedSqlExtensions.cs

```diff
@@ -45,4 +45,11 @@ public static ParameterizedSql AsUnrolledSqlInExpression<T>(this IEnumerable<T>
             0 => SQL($"(null)"),
             _ => SQL($"(").Append(set.Select(item => SQL($"{AsSqlParam(item)}")).ConcatenateSql(SQL($","))).Append(SQL($")")),
         };
+
+    /// <summary>
+    /// Generate an unique name which can be used safely as an alias for the given query.
+    /// </summary>
+    [Pure]
+    public static ParameterizedSql GenerateUniqueQueryAlias(this ParameterizedSql query)
+        => ParameterizedSqlSubQueries.GenerateUniqueQueryAlias(query.CommandText().GetHashCode(), query.CommandText());
```

Comment:
Ik zou er ook mee kunnen leven als de seed verplicht is; dit via GetHashCode doen is vermoedelijk niet echt nodig en iig een implementatie-detail.  Maar zo mag zeker ook.